tools/ci_build/github/mac-ios-swift-packaging-pipeline.yml

```
@@ -0,0 +1,117 @@
+parameters:
+- name: BuildType
+  displayName: |-
+    Type of build.
+    "normal": A normal build not for publication.
+  type: string      #TODO: add other build types here
+  values:
+  - normal
+  default: normal
+
+name: "$(Date:yyyyMMdd)$(Rev:rrr)"  # build number format
+
+stages:
+- stage: SPM_IOS_Local_Packaging_Testing
+  dependsOn: []
+  jobs:
+  - job: SPMIosPackaging
+    displayName: "SPM iOS Packaging Local"
```

```suggestion
    displayName: "Test with local ORT native pod"
```

Any CI here is 'SPM' and 'iOS' so I think that can be assumed.